ccan/ccan/str/hex/hex.c

```diff
@@ -64,3 +64,21 @@ bool hex_encode(const void *buf, size_t bufsize, char *dest, size_t destsize)
 
 	return true;
 }
+
+bool hex_encode_reversed(const void *buf, size_t bufsize, char *dest, size_t destsize)
```

Comment:
OK, so ccan is an external library, so we try not to make changes here; we do it there and reimport.

In this case, reversed hex is so whack that I think we're better off carrying our own wrapper.